samples/buildSrc-test-junit4/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+import tools.sayHelloTo
+
+tasks {
```

Comment:
For a single task the `TaskScope` is a little bit out of bounds.
I would recommend to use either `val hello by tasks.creating {}` or directly `tasks.register("hello") {}`..

Reply:
done